playbooks/vars/openstack-service-config.yml

```diff
@@ -72,7 +72,6 @@ openstack_images:
       hw_disk_bus: "scsi"
       hw_vif_multiqueue_enabled: "true"
       hw_qemu_guest_agent: "yes"
-      hypervisor_type: "qemu"
```

Comment:
why was this taken out?  iirc we need it to ensure nova doesn't try to schedule these images to ironic nodes...

Reply:
in master / queens qemu != kvm

Reply:
ok, so should we change this to kvm?

Reply:
done